src/options/options.css

```
@@ -1,7 +1,11 @@
+:root {
+  --gray: rgba(12, 12, 13, 0.15);
```

Or, different idea, maybe rename it for the actual use case?

```suggestion
  --gray-separator: rgba(12, 12, 13, 0.15);
```